### ISSUE TITLE
Feature request: quick keyword access to bookmarks

### ISSUE THREAD

**Issue by Sh4d1** [tridactyl/tridactyl#4731]
Firefox allows you to add keywords to bookmarks, and if you have one, let's say gh for Github.com, then when I type gh<Enter> in the classic URL bar, it sends me directly to the site, whereas it does not work that way in the tridactyl bar.

It would be nice to support that in `:open` and `:tabopen`

**Comment by oredaze** [comment]
It's not quite the same, but I would like to point to `set searchurls`. I have a search engine for github (keyword `gh`) and if I type `tabopen gh` but no search after that it takes me to the root website.

**Comment by Sh4d1** [comment]
Yeah I saw that, but it's a bit annoying to add each quick bookmark I have to a searchurl :sweat_smile:

**Comment by oredaze** [comment]
An other option I use is `bind ga tabopen https://artstation.com`. You can have all your bookmarks under one hotkey (g for me) like that.

**Comment by bovine3dom** [comment]
Does e.g. `:set bmarkweight 10000000` do what you want?

Direct support for bookmark keywords was ruled out by Mozilla a long time ago - see #73.

> it's a bit annoying to add each quick bookmark I have to a searchurl

You can bulk import them using gholk's script here:

https://github.com/tridactyl/tridactyl/wiki/Import-bookmarks-with-keyword-from-exported-firefox-bookmarks
